testing/compose_files/docker-compose-presto.yml

```diff
@@ -47,13 +47,13 @@ services:
       - ./../../testing/metastore/db-data:/var/lib/hive
       - ./../../testing/hadoop/hadoop-dfs-name:/hadoop/dfs/name
       - ./../../testing/hadoop/hadoop-dfs-data:/hadoop/dfs/data
-      - ./../../testing/hadoop/hadoop-logs:/opt/hadoop/logs
+#      - ./../../testing/hadoop/hadoop-logs:/opt/hadoop/logs
```

Comment:
Why is this commented?

Reply:
I had a lot of problems getting that to function properly and consistently. Please check out the [ubi-hadoop Dockerfile](https://github.com/RedHatInsights/ubi-hadoop/blob/main/Dockerfile) and [ubi-hive Dockerfile](https://github.com/RedHatInsights/ubi-hive/blob/main/Dockerfile). If there's some secret sauce for this, I'd definitely like to know.

The issue I had was that the mount would change from being owned by `hadoop` to `root`. Java could not write to the dir and the Java VM would not start and the container would exit on error.